docs/test/test.html

```diff
@@ -5,6 +5,7 @@
     <meta charset="UTF-8">
 </head>
 <body>
-    <script src="test.js"></script>
+    <script src="featherRunner.js"></script>
+    <script src="featherSpecs.js"></script>
```

Comment:
This was automatically added when I ran `npm run test`. Not sure where it came from

Reply:
that's fine